src/wp-admin/includes/ajax-actions.php

```diff
@@ -3970,23 +3970,48 @@ function wp_ajax_crop_image() {
 			/** This filter is documented in wp-admin/includes/class-custom-image-header.php */
 			$cropped = apply_filters( 'wp_create_file_in_uploads', $cropped, $attachment_id ); // For replication.
 
-			$parent_url = wp_get_attachment_url( $attachment_id );
-			$url        = str_replace( wp_basename( $parent_url ), wp_basename( $cropped ), $parent_url );
+			$parent_url      = wp_get_attachment_url( $attachment_id );
+			$parent_basename = wp_basename( $parent_url );
+			$url             = str_replace( $parent_basename, wp_basename( $cropped ), $parent_url );
 
 			$size       = wp_getimagesize( $cropped );
 			$image_type = ( $size ) ? $size['mime'] : 'image/jpeg';
 
+			// Get the original image's post to pre-populate the cropped image.
+			$original_attachment      = get_post( $attachment_id );
+			$sanitized_post_title     = sanitize_file_name( $original_attachment->post_title );
+			$use_original_title       = (
+				'' !== trim( $original_attachment->post_title ) &&
```

Comment:
```suggestion
				( '' !== trim( $original_attachment->post_title ) ) &&
```
for consistency with the other checks.